vip-jetpack/connection-pilot/class-jetpack-connection-pilot.php

```diff
@@ -21,18 +25,10 @@ class Connection_Pilot {
 	 */
 	const CRON_ACTION = 'wpcom_vip_run_jetpack_connection_pilot';
 
-	/**
-	 * The schedule the cron job runs on. Update in 000-vip-init.php as well.
-	 *
-	 * Schedule changes can take up to 24 hours to take effect.
-	 * See the a8c_cron_control_clean_legacy_data event for more details.
-	 */
-	const CRON_SCHEDULE = 'hourly';
```

Comment:
Why do we change this constant to a string literal?

Reply:
Only used in one-place, and I figured moving closer to it's source would make future devs more aware that changing the schedule requires more thought than just updating the const.